app/renderer/lib/appium-client.js

```diff
@@ -315,26 +315,44 @@ export default class AppiumClient {
           }
         }
       } else if (this.driver.client.isIOS) {
-        // on iOS, find the top status bar and address bar and use its Y endpoint
-        const topBar = await this.fetchElement({
-          strategy: '-ios class chain',
-          selector: IOS_TOP_CONTROLS_SELECTOR,
-        });
-        if (topBar.el) {
-          const {y, height} = await topBar.el.getRect();
-          webviewTopOffset = y + height;
-        }
-        // in landscape mode, there is empty space on both sides (at default zoom level), so add offset for that too
-        if (windowSize.height < windowSize.width) {
-          try {
-            const deviceScreenInfo = await this.driver.executeScript('mobile:deviceScreenInfo', []);
-            webviewLeftOffset = deviceScreenInfo.statusBarSize.height;
-          } catch (e) {
+        const isSafari = this.driver.client?.capabilities?.browserName?.toLowerCase() === 'safari';
```

Comment:
I believe CI is failing because Parcel 1 doesn't support optional chaining. Maybe it's worth adding it as a prop in [`main/app/renderer/reducers/Inspector.js`](https://github.com/appium/appium-inspector/blob/main/app/renderer/reducers/Inspector.js) the same way as `automationName`?

Reply:
yeah just noticed this. i went about the problem in a slightly different way, what do you think

Reply:
That works too :)